Things.md

```diff
@@ -169,4 +170,10 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing classes
+
+Classes are implemented using [ObjectMakr](https://github.com/FullScreenShenanigans/ObjectMakr), a module that automates the creation of the inheritance structure of classes. The structure and a list of properties are given and the classes can then be constructed. 
```

Comment:
> The structure... ...can then be constructed.
> 

It would be a bit more clear to specify that this is when it's constructed. Now it's ambiguous when that happens.